stl/inc/chrono

```diff
@@ -1480,6 +1481,33 @@ namespace chrono {
         return _Result;
     }
 
+    template <class _ToPeriod, class _FromPeriod>
+    constexpr bool _Duration_cast_would_underflow = _FromPeriod::den > INTMAX_MAX / _ToPeriod::num;
+
+    template <class _From, class _To>

```

Comment:
This uses the order "from, to" while everything else related to `duration_cast` uses the order "to, from", which seems to be confusing (although all of the usage appears to be correct).

Reply:
It's because this was a concept. I think it ends up nicer as a simple variable though.

Reply:
I just folded this into the following variable, less instantiations to worry about too.